src/lv_message_efficient.cc

```diff
@@ -204,27 +193,38 @@ namespace grpc_labview
     {
         for (auto nestedMessage : _repeatedMessageValuesMap)
         {
+            if (!nestedMessage.second.get()->_validData)
```

Comment:
NOTE: This check ensures we only copy the data into a LV data structure once. In particular, it handles cases where the message element at index zero in the array had a non-default value for a repeated message field but the message element at index one had an empty array. In that case, we don't want to copy the array from the previous element again.